linux/update_vcpkg_manager.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -e
+
+if [ ! -d "linux" ]; then
+    echo "start this script in the source root directory"
+    exit 1
+fi
+
+CACHE_DIR="$PWD/3rdParty/buildCache/linux"
+BUILD_DIR="$PWD/3rdParty/linux"
+VCPKG_PORTS="$PWD/3rdParty/vcpkg_ports"
+VCPKG_ROOT="$BUILD_DIR/vcpkg"
+
+if [ ! -d $VCPKG_ROOT ]; then
+    mkdir -p $BUILD_DIR
+    git -C $BUILD_DIR clone https://github.com/JackBoosY/vcpkg -b dev/jack/23758
```

Comment:
Build is probably failed because of this custom repo that is probably outdated and doesn't contain the link to the latest release of vcpkg-tool

Reply:
I know. I already handle it.